app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt

```diff
@@ -18,8 +19,8 @@ data class BridgeFee(
     val name: String,
     val decimals: Int
 ) : Parcelable {
-    val amountInToken
-        get() = amount?.toBigDecimal()?.orZero()?.divide(decimals.toPowerValue()).orZero()
+    val amountInToken: BigDecimal
+        get() = amount?.toBigDecimal()?.let { it.divide(decimals.toPowerValue()) }.orZero()
```

Comment:
I know this isn't part of the code you worked on, but decimal division without rounding and specified scaling might cause either `ArithmeticException` or incorrect calculations. 
```suggestion
        get() = amount.toBigDecimalOrZero().divideSafe(decimals.toPowerValue())
```